EDK-II-Development-Process.md

```diff
@@ -1,297 +1,222 @@
 
-      - Add the `--cover-letter` parameter for long patch series. (Be
-        sure to edit the cover-letter.)
+      - The pull request fails the `PatchCheck.py` check. Resolve the issues reported by `PatchCheck.py`. A force push
+        to `<new-integration-branch>` can be used to automatically restart the checks.
 
-      - Add the `--subject-prefix="PATCH v2"` if you are sending out a
-        second version of the patch series.
+      - The pull request fails Windows or Ubuntu checks. The GitHub pull request provides links to the Azure Pipelines
+        results that are used to view test results for checks that failed.
 
-    - `$ git send-email *.patch --to devel@edk2.groups.io`
+    - If you are unsure how to resolve a PR failure, leave a comment in the PR to ask for help.
 
-    - If it is the first time to send mail to edk2 mail list, please join
-      https://edk2.groups.io/g/devel and expect there will be delay because
-      the mail needs manual approval from the admin (gaoliming@byosoft.com.cn
-      or michael.d.kinney@intel.com) of https://edk2.groups.io/g/devel
+12. Modify local commits based on the review feedbacks and repeat steps 2 to 10.
 
-13. Modify local commits based on the review feedbacks and repeat steps
-    3 to 11
+    - For the latest commit, you can use `>$ git commit --amend`
 
-    - For the latest commit, you can use `$ git commit --amend`
-
-    - For multiple commits use `$ git rebase -i origin/master`
+    - For multiple commits use `>$ git rebase -i origin/master`
 
     - Consult your git gurus on edk2-devel or irc channel if you have
       questions.
 
-The maintainer process for the EDK II project
----------------------------------------------
+13. Update [TianoCore Bugzilla](https://bugzilla.tianocore.org/) issue(s) resolved by the change(s).
 
-1.  Determine if a patch has met the review requirements for the package
+    - Add a pointer to the GitHub pull request (e.g. [Readme.md: Update EDK II CI build branch name](https://github.com/tianocore/edk2/pull/153)).
 
-2.  Update the origin/master from the server
+    - Add the PR merged to `edk2/master` (e.g. `Completed in PR https://github.com/tianocore/edk2/pull/153`).
 
-    `$ git fetch origin`
+    - Mark BZ issues as Resolved/Fixed.
 
-3.  Create and checkout an integration branch
+## Maintainer Process
 
-    `$ git checkout -b <new-integration-branch> origin/master`
+1. It is recommended to register for email notifications for pull requests, pushes, and check status results by
+   setting up notifications for the EDK II repository ([tianocore/edk2](https://github.com/tianocore/edk2)).
 
-4.  Add commits on the integration branch
+   - [Configuring notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications)
 
-    `$ git am <patch-file>`
+2. Verify that the Pull Request title and description succinctly describe the changes.

```

Comment:
Mention here that PR descriptions are similar to `PATCH 0` (cover letter) in the previous process.

Reply:
Done